web/app/view/dialog/User.js

```diff
@@ -175,6 +175,31 @@ Ext.define('Traccar.view.dialog.User', {
                     }
                 }
             }]
+        }, {
+            xtype: 'fieldset',
+            title: Strings.twofa,
+            collapsible: true,
+            collapsed: true,
+            items: [{
+                xtype: 'checkboxfield',
+                inputValue: true,
+                uncheckedValue: false,
+                name: 'useTotp',
+                fieldLabel: Strings.useTotp,
+                reference: 'useTotpField',
+                handler: 'useTotp'
+            }, {
+                xtype: 'displayfield',
+                name: 'totpKey',
+                reference: 'totpKeyField',
+                fieldLabel: Strings.totpKey
+            }, {
+                xtype: 'button',
```

Comment:
I will look into this. First time working with ExtJS so still reading the docs on how I can do this.

Reply:
I was able to fix this.
<img width="348" alt="Screenshot 2019-06-03 at 14 09 12" src="https://user-images.githubusercontent.com/36760980/58800850-43b0d200-8609-11e9-976f-c3ac07d598d4.png">

Reply:
It doesn't look like what we have for token. Can you please make UI consistent with what we already have.

Reply:
Ok I was able to get it look just like Token field.

<img width="353" alt="Screenshot 2019-06-05 at 08 41 36" src="https://user-images.githubusercontent.com/36760980/58935531-f8fc9a80-876d-11e9-9046-7ae28d597371.png">

Do you stil want met to ad the two field to the 'Permissions' fieldset or is it ok to have them in their own '2FA' fieldset?

Reply:
Yes, I want it under permissions. Also, I hate to say this, but it still doesn't look the same to me.

Reply:
It is now in the permissions fieldset. Also the width of the input field it the same as the ones above. In the previous I made it a bit wider because it would show all characters in the key. Now a small part is hidden behing the icon.

<img width="340" alt="Screenshot 2019-06-19 at 15 44 22" src="https://user-images.githubusercontent.com/36760980/59770965-2b97ae80-92a9-11e9-87c7-03245e5f834d.png">